src/application-constants.js

```diff
@@ -40,8 +40,8 @@ function windowDefaults () {
   const messageHeight = 100
   return {
     bounds: {
-      width: 500,
-      height: headerHeight + messageHeight * 6
+      width: 1100,
+      height: headerHeight + messageHeight * 8
```

Comment:
What's about this change?

Reply:
If we set the default width to 500px the titles of chats might get truncated which is bad for testing. I also see no reason why we set the default width to 500, that's even less than a mobile device would have...